tests/integration/targets/vmware_vcenter_settings/aliases

```diff
@@ -1,5 +1,3 @@
-shippable/vcenter/group2
 cloud/vcenter
-needs/target/prepare_vmware_tests
+unsupported
```

Comment:
Why did you add `usupported`?

Reply:
Thank you for pointed out.
Sorry, I think this line is wrong, and I fixed it.